nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,583 @@
+    )
+
+    wm_annot = traits.Tuple(
+        traits.String,
+        traits.Int,
+        traits.Int,
+        argstr="--wm-annot %s %i %i",
+        desc="annot lhbase rhbase : annotation to use for WM segmentation (with --subsegwm, default is lobes 3200 4200)",
+    )
+
+    output_upsampling_factor = traits.Int(
+        argstr="--output-usf %i",
+        desc="set output USF different than USF, mostly for debugging",
+    )
+
+    head = traits.String(
+        argstr="--head %s", desc="use headseg instead of apas+head.mgz"
+    )
+
+    subseg_cblum_wm = traits.Bool(
+        argstr="--subseg-cblum-wm", desc="subsegment cerebellum WM into core and gyri"
+    )
+
+    no_pons = traits.Bool(
+        argstr="--no-pons", desc="do not add pons segmentation when doing ---xcerseg"
+    )
+
+    no_vermis = traits.Bool(
+        argstr="--no-vermis",
+        desc="do not add vermis segmentation when doing ---xcerseg",
+    )
+
+    ctab = File(exists=True, argstr="--ctab %s", desc="colortable")
+    no_seg_stats = traits.Bool(
+        argstr="--no-seg-stats", desc="do not compute segmentation stats"
+    )
+
+
+class GTMSegOutputSpec(TraitedSpec):
+    out_file = File(desc="GTM segmentation")
+
+
+class GTMSeg(FSCommand):
+    """create an anatomical segmentation for the geometric transfer matrix (GTM).
+
+    Examples
+    --------
+    >>> gtmseg = GTMSeg()
+    >>> gtmseg.inputs.subject_id = 'subject_id'
+    >>> gtmseg.cmdline
+    'gtmseg --o gtmseg.mgz --s subject_id'
+    """
+
+    _cmd = "gtmseg"
+    input_spec = GTMSegInputSpec
+    output_spec = GTMSegOutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        outputs['gtm_file'] = os.path.join(
+            self.inputs.subjects_dir, self.inputs.subject_id, 'mri', self.inputs.out_file
+        )
+        return outputs
+
+
+class GTMPVCInputSpec(FSTraitedSpec):
+
+    in_file = File(
+        exists=True,
+        argstr="--i %s",
+        mandatory=True,
+        copyfile=False,
+        desc="input volume - source data to pvc",
+    )
+
+    frame = traits.Int(
+        argstr="--frame %i", desc="only process 0-based frame F from inputvol"
+    )
+
+    psf = traits.Float(argstr="--psf %f", desc="scanner PSF FWHM in mm")
+
+    segmentation = File(
+        argstr="--seg %s",
+        exists=True,
+        mandatory=True,
+        desc="segfile : anatomical segmentation to define regions for GTM",
+        xor=["reg_file", "regheader"]
+    )
+
+    reg_file = File(
+        exists=True,
+        argstr="--reg %s",
+        mandatory=True,
+        desc="LTA registration file that maps PET to anatomical",
+    )
+
+    regheader = traits.Bool(
+        argstr="--regheader", desc="assume input and seg share scanner space"
+    )
+
+    reg_identity = traits.Bool(
+        argstr="--reg-identity", desc="assume that input is in anatomical space"
+    )
```

Comment:
I *think* this is what we want. It's what I was suggesting, anyway. Apologies for the confusion.

```suggestion
    )

    _reg_xor=["reg_file", "regheader", "reg_identity"]
    reg_file = File(
        exists=True,
        argstr="--reg %s",
        mandatory=True,
        desc="LTA registration file that maps PET to anatomical",
        xor=_reg_xor,
    )

    regheader = traits.Bool(
        argstr="--regheader",
        desc="assume input and seg share scanner space",
        xor=_reg_xor,
    )

    reg_identity = traits.Bool(
        argstr="--reg-identity",
        desc="assume that input is in anatomical space",
        xor=_reg_xor,
    )
```

Reply:
Thanks @effigies!